.github/workflows/project-automation.yml

```
@@ -0,0 +1,43 @@
+                card_id: getCardId(pr.number),
+                position: 'bottom',
+                column_id: 'Cancelled'
+              });
+            } else if (pr.state === 'open') {
+              // If PR was opened/reopened
+              await github.rest.projects.moveCard({
+                card_id: getCardId(pr.number),
+                position: 'bottom',
+                column_id: 'In Progress'
+              });
+            }
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)